test/hank_test_utils.erl

```diff
@@ -21,9 +21,16 @@ init() ->
             rebar_state:new()),
     State.
 
+init(AppName) when is_atom(AppName) ->
+    RebarAppInfo =
+        rebar_app_info:name(
+            rebar_app_info:new(), atom_to_binary(AppName, utf8)),
+    RebarAppInfo2 = rebar_app_info:dir(RebarAppInfo, abs_test_path(atom_to_list(AppName))),
+    rebar_state:project_apps(init(), RebarAppInfo2).
+
 mock_context(Apps) ->
     AppsAbs = maps:map(fun(_App, Path) -> filename:absname(Path) end, Apps),
-    hank_context:new(AppsAbs).
+    hank_context:new(AppsAbs, [test_app]).
```

Comment:
It shouldn't be `hank_context:new(AppsAbs, Apps).`?
The hardcoded `[test_app]` is suspicious.